lldb/tools/lldb-dap/package.json

```diff
@@ -255,6 +255,11 @@
 								"type": "string",
 								"description": "The escape prefix to use for executing regular LLDB commands in the Debug Console, instead of printing variables. Defaults to a back-tick (`). If it's an empty string, then all expression in the Debug Console are treated as regular LLDB commands.",
 								"default": "`"
+							},
+							"customFrameFormat": {
+								"type": "string",
+								"description": "If non-empty, stack frames will have descriptions generated based on the provided format. See https://lldb.llvm.org/use/formatting.html for an explanation on format strings for frames. If the format is wrong, the default display names for frames will be used. This might come with a performance cost because debug information might need to be processed to generate the description.",
```

Comment:
Instead of saying "If the format is wrong", maybe say "If the format string contains errors an error string will be emitted into the debug console and the default frame names will be used".